packages/webapp/public/locales/en/translation.json

```diff
@@ -777,7 +777,7 @@
     "WHOLE_FARM_REVENUE": "Whole farm revenue"
   },
   "FORM_VALIDATION": {
-    "OVER_200_CHARS": "Text cannot exceed 200 characters"
```

Comment:
I realized that Mwaya is using this in his [PR](https://github.com/LiteFarmOrg/LiteFarm/pull/2471/files). Should we keep both 200 and 255 characters?

Reply:
I asked Kevin about this 200 char limit and he said it was arbitrary and that we could go up to 255 if we want to. 

Since 255 is the database limit it seems natural to use the limit and just remove the 200 I think. I can let Mwaya know though!